README.md

```diff
@@ -57,6 +64,7 @@ http://guidesmiths-react-form-builder.s3-website.eu-central-1.amazonaws.com/
 |   onSubmit*    |   Action to be realised "onSubmit" form    |    function       |   -    |
 |   language	| Shortcut with the language  to render components in multiple languages (`country`,`date`) <br /> <br /> Available laguages: `es`,`de`,`fr`,`en`  	| string   	|   en	|
 |  isoCode      |   Isocode of the country to show as default in phone input |  string     | GB
+|  isMobile      |   Checked component by default | boolean    | false
```

Comment:
may be we can change it to have a better description? Something more specific of the flag. E.g:
`A boolean flag to determine if the we're from a mobile port view`